.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: "08:00"
+      timezone: "Europe/Berlin"
+    open-pull-requests-limit: 5
+
+  - package-ecosystem: "composer"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: "08:00"
+      timezone: "Europe/Berlin"
+    open-pull-requests-limit: 5
+    groups:
+      composer-prod-updates:
+        update-types:
+          - "minor"
+          - "patch"
```

Comment:
@m-seidel How are major versions are handled?

On the Product website, we're using this action on a monthly base.